esmvaltool/cmorizers/obs/download_scripts/download_cds_satellite_soil_moisture.py

```diff
@@ -0,0 +1,150 @@
+"""
+Download cds-satellite-soil-moisture.
+
+Script to download cds-satellite-soil-moisture
+from the Climate Data Store(CDS).
+"""
+
+import argparse
+import os
+import tarfile
+
+import yaml
+
+import cdsapi
+
+
+def download_cds_satellite_sm():
+    """Download dataset cds-satellite-soil-moisture."""
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--config_file',
+                        '-c',
+                        default=os.path.join(os.path.dirname(__file__),
+                                             'config-user.yml'),
+                        help='config file',
+                        required=True)
+    parser.add_argument('--frequency',
+                        '-f',
+                        default=None,
```

Comment:
It's not clear what happens with `None`, I would set the default to `monthly` or don't set it at all.